src/drivers/gps/gps.cpp

```diff
@@ -535,11 +535,12 @@ void GPS::handleInjectDataTopic()
 
 	bool updated = false;
 
-	// Limit maximum number of GPS injections to 6 since usually
+	// Limit maximum number of GPS injections to 12 since usually
 	// GPS injections should consist of 1-4 packets (GPS, Glonass, BeiDou, Galileo).
 	// Looking at 6 packets thus guarantees, that at least a full injection
 	// data set is evaluated.
-	const size_t max_num_injections = 6;
+	// Moving Base reuires a higher rate, so we allow up to 12 packets.
+	const size_t max_num_injections = 12;
```

Comment:
The queue depth for gps_inject_data is only 8.

Reply:
I'll reduce to 8.

Reply:
Seeing 50Hz injection rate with moving baseline.